bin/make_glossary_sections.pl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl -w
```

Comment:
Script for writing glossary with section headers. Alternative to using glossary directive.